src/Services/Air/AirParser.js

```diff
@@ -925,6 +925,11 @@ function importRequest(data) {
   return response;
 }
 
+function universalRecordRetrieveRequest(data) {
+  const response = extractBookings.call(this, data);
```

Comment:
Have you tried calling this request with booking which was not imported before retrieval? What will be the behaviour of this parser?

Reply:
It will raise RuntimeError.AirRuntimeError "Record locator not found", which I thought should be adequate, should it return an empty array?